cypress/test/scripts/tools/extract-tools-github.cy.js

```diff
@@ -0,0 +1,199 @@
+                                        "api",
+                                        "code-first",
+                                        "code-generator",
+                                        "converter",
+                                        "directory",
+                                        "documentation-generator",
+                                        "editor",
+                                        "ui-component",
+                                        "dsl",
+                                        "framework",
+                                        "github-action",
+                                        "mocking-and-testing",
+                                        "validator",
+                                        "compare-tool",
+                                        "other",
+                                        "cli",
+                                        "bundler",
+                                        "ide-extension"
+                                    ]
+                                },
+                                {
+                                    "type": "string"
+                                }
+                            ]
+                        },
+                        "minItems": 1,
+                        "examples": [
+                            "api",
+                            "code-first",
+                            "code-generator",
+                            "converter",
+                            "directory",
+                            "documentation-generator",
+                            "editor",
+                            "ui-component",
+                            "dsl",
+                            "framework",
+                            "github-action",
+                            "mocking-and-testing",
+                            "validator",
+                            "compare-tool",
+                            "other",
+                            "cli",
+                            "bundler",
+                            "ide-extension"
+                        ]
+                    },
+                    "hasCommercial": {
+                        "type": "boolean",
+                        "description": "Indicate if your tool is open source or commercial offering, like SAAS for example",
+                        "default": false
+                    }
+                }
+            }
+        };
+
+        // Intercept the API request and stub the response
+        cy.intercept('GET', 'https://api.github.com/search/code?q=filename:.asyncapi-tool*', {
+            statusCode: 200,
+            body: stubbedResponse,
+            headers: {
+                accept: 'application/vnd.github.text-match+json',
+                authorization: `token *`,
+            },
+        }).as('getData');
+
+        // Manually trigger the function
+        await getData().then((response) => {
+            expect(response).to.exist;
```

Comment:
response returned in this line is not correct. Kindly update the `stubbedResponse` and recheck the response.